### PR TITLE
Fixes #18584: semaphore gurading LDAP repos are created each time

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/18584

### CODE REVIEW

File: webapp/sources/api-doc/paths/nodes/id.yml [hunk 1]
```diff
@@ -102,6 +102,16 @@ delete:
   operationId: deleteNode
   parameters:
     - $ref: ../../components/parameters/node-id.yml
+    - name: "mode"
+        in: query
+        required: false
+        description: Deletion mode to use, either move to trash ('move', default) or erase ('erase')
+        schema:
```
Comment (amousset) [inline webapp/sources/api-doc/paths/nodes/id.yml:109]: I guess there is a default for compatibility, it should be documented with:

```yaml
default: move
```

**Comment by fanf** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/paths/nodes/id.yml (hunk 1 above)
Comment (fanf) [reply webapp/sources/api-doc/paths/nodes/id.yml:109]: corrected in PR #3347

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**